src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -928,9 +938,10 @@ private void PreparePlainExecutables()
         }
     }
 
-    private void PrepareProjectExecutables()
+    private async Task PrepareProjectExecutablesAsync(CancellationToken cancellationToken)
     {
         var modelProjectResources = _model.GetProjectResources();
+        var dcpInfo = await dcpDependencyCheckService.GetDcpInfoAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
Perfect! I need this information to get the container runtime 😄 for the host name in this file.